siriuspy/siriuspy/timesys/ll_classes.py

```diff
@@ -678,7 +678,7 @@ def _set_source(self, value):
             dic_ = {'Src': n}
         else:
             n -= offset
-            evt = int(_TIConst.EvtHL2LLMap[pname][-2:])
+            evt = int(_TIConst.EvtHL2LLMap[pname][3:])
```

Comment:
Could we use some strip function for letters and symbols and leave just numbers instead? That seems like it would document the intent better.

Reply:
done!